### ISSUE TITLE
 Can't resolve './PickerAndroid',  Can't resolve './PickerIOS' in picker/js

### ISSUE THREAD

**Issue by syirilrakhulh** [react-native-picker/picker#82]
i getting error when importing Picker from "@react-native-community/picker". The error say can't resolve PickerAndroid.js and PickerIOS.js in /picker/js. when i checked, yes it's nothing. are u forget to move to js or i wrong with usage way? in docs nobody tell about PickerAndroid.js. Somebody can tell me? thank u 

my package.json is 
{
  "main": "index.js",
  "scripts": {
    "android": "react-native run-android",
    "ios": "react-native run-ios",
    "web": "expo start --web",
    "start": "react-native start",
    "test": "jest"
  },
  "dependencies": {
    "@react-native-community/masked-view": "^0.1.10",
    "@react-native-community/picker": "^1.5.0",
    "@react-navigation/bottom-tabs": "^5.4.5",
    "@react-navigation/native": "^5.4.0",
    "date-fns": "^2.13.0",
    "expo": "~37.0.3",
    "expo-splash-screen": "^0.2.3",
    "expo-updates": "~0.2.0",
    "react": "~16.9.0",
    "react-dom": "~16.9.0",
    "react-native": "~0.61.5",
    "react-native-gesture-handler": "^1.6.1",
    "react-native-reanimated": "^1.8.0",
    "react-native-safe-area-context": "^1.0.0",
    "react-native-screens": "^2.7.0",
    "react-native-timeline-flatlist": "^0.7.2",
    "react-native-unimodules": "~0.9.0",
    "react-native-web": "~0.11.7"
  },
  "devDependencies": {
    "@babel/core": "~7.9.0",
    "babel-jest": "~25.2.6",
    "jest": "~25.2.6",
    "react-test-renderer": "~16.9.0"
  },
  "jest": {
    "preset": "react-native"
  },
  "private": true
}

**Comment by me301** [comment]
I'm facing the same issue. Any solutions?

**Comment by 19neloyk** [comment]
I am having the same issue, but everything renders correctly when I simulate it on the iPhone

**Comment by tomas-hartman** [comment]
Similar issue [here](https://user-images.githubusercontent.com/44980733/84590114-0c881a00-ae34-11ea-8b62-989f800f4816.png).

It tells me that `index.js` does not exist, even though I found it pretty much in the location. Fails on Android, couldn't try on iOS.

**Comment by smontiu** [comment]
error: Error: Unable to resolve module `react-native-web/dist/modules/UnimplementedView` from `node_modules/@react-native-picker/picker/js/PickerIOS.js`: react-native-web/dist/modules/UnimplementedView could not be found within the project.

"@react-native-picker/picker@^1.8.3": version "1.9.5"

The problem comes from the library: "react-native-picker-select" with "8.0.4" seems to fail.
"react": "16.13.1"
"react-native": "0.63.4"

Try to use a lower version

**Comment by LiuTangLei** [comment]
> error: Error: Unable to resolve module `react-native-web/dist/modules/UnimplementedView` from `node_modules/@react-native-picker/picker/js/PickerIOS.js`: react-native-web/dist/modules/UnimplementedView could not be found within the project.
> 
> "@react-native-picker/picker@^1.8.3": version "1.9.5"
> 
> The problem comes from the library: "react-native-picker-select" with "8.0.4" seems to fail.
> "react": "16.13.1"
> "react-native": "0.63.4"
> 
> Try to use a lower version, with "8.0.3" works fine

Thanks, work for me

**Comment by ssthil** [comment]
error: Error: Unable to resolve module `react-native-web/dist/modules/UnimplementedView` from `node_modules/@react-native-picker/picker/js/PickerIOS.js`: react-native-web/dist/modules/UnimplementedView could not be found within the project.

My env:
    "@react-native-picker/picker": "1.9.5",
    "react-native-picker-select": "8.0.3",

     "react": "16.13.1",
    "react-native": "0.63.2",

![image](https://user-images.githubusercontent.com/3829988/102500632-cf1f7600-407c-11eb-8d2d-b435f2272f20.png)

@smontiu could you please advise me?

**Comment by smontiu** [comment]
@ssthil can you double check, removing node_modules, yarn.lock and yarn install again ?

**Comment by ivelaval** [comment]
@syirilrakhulh @ssthil 

The problem is because of a upgrade from `@react-native-community/picker` [https://www.npmjs.com/package/@react-native-community/picker ] basically this package changed their name to `@react-native-picker/picker` [https://github.com/react-native-picker/picker] and for that reason the package `react-native-picker-select`  should have updated the package and implement a fix for this change in `8.0.4`.

**Solution**
Keep `8.0.4` version and install **@react-native-picker/picker** and that's it.

**Possible issue in Android Studio**
If you have problems in Android Studio execute file > sync project with Gradle files.

Let me know if it's working for you

**Comment by jackywxd** [comment]
I have the same problem when building my project with GitHub action, but it works fine in my iMac. So this seems only happening in Linux, not in Mac OS.

I tried both 8.0.3 and 8.0.4 react-native-picker-select, but same error.

```
1
Run npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res
2
  npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res
3
  shell: /bin/bash -e {0}
4
  env:
5
    JAVA_HOME_8.0.275_x64: /opt/hostedtoolcache/jdk/8.0.275/x64
6
    JAVA_HOME: /opt/hostedtoolcache/jdk/8.0.275/x64
7
    JAVA_HOME_8_0_275_X64: /opt/hostedtoolcache/jdk/8.0.275/x64
8
                 Welcome to React Native!
9
                Learn once, write anywhere
10

11

12
error Unable to resolve module `react-native-web/dist/modules/UnimplementedView` from `node_modules/@react-native-picker/picker/js/PickerIOS.js`: react-native-web/dist/modules/UnimplementedView could not be found within the project.
13
Error: Unable to resolve module `react-native-web/dist/modules/UnimplementedView` from `node_modules/@react-native-picker/picker/js/PickerIOS.js`: react-native-web/dist/modules/UnimplementedView could not be found within the project.
14
```
